### PR TITLE
Fix appendices sections

### LINKED ISSUE #598: IANA consideration should be its own section
At present the IANA consideration is section B.10, part of Appendix B "Frequently Asked Questions". It should be the separate Appendix "C". 

(Actually, but that is a matter of taste, I would reverse the order and place the IANA section as Appendix B.)

### PR DESCRIPTION
**Pull request by rhiaro**
Fixes #598

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/603.html" title="Last updated on Feb 12, 2021, 2:27 PM UTC (1bf5b38)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/603/1dda7f5...1bf5b38.html" title="Last updated on Feb 12, 2021, 2:27 PM UTC (1bf5b38)">Diff</a>

### CODE REVIEW

**Review by iherman — APPROVED** [review]

**Review by talltree — APPROVED** [review]
Thanks Amy

File: index.html [hunk 1]
```diff
@@ -6313,6 +5979,337 @@ <h2>application/did+dag+cbor</h2>
...
+          " >
+          <figcaption>
+The <a>DID subject</a> is the same entity as the <a>DID controller</a>
+          </figcaption>
+        </figure>
+        <p>
+From a graph model perspective, even though the nodes identified as the
+<a>DID controller</a> and <a>DID subject</a> in figure 4 are distinct,
+there is a logical arc connecting them to express a semantic equivalence
+relationship (in RDF/OWL, this is expressed using the <a href="https://www.w3.org/TR/owl-ref/#sameAs-def"><code>owl:sameAs</code> predicate)</a>.
```
Comment (TallTed) [inline index.html:6224]:
```suggestion
relationship (in RDF/OWL, this is expressed using the <a
href="https://www.w3.org/TR/owl-ref/#sameAs-def">
<code>owl:sameAs</code> predicate)</a>.
```

File: index.html [hunk 2]
```diff
@@ -6313,6 +5979,337 @@ <h2>application/did+dag+cbor</h2>
...
+        </p>
+        <figure id="group-did-controllers">
+          <img style="margin: auto; display: block; width: 75%;"
+            src="diagrams/figure-c.2-group-did-controllers.png" alt="
+            Diagram showing how three DID controllers act together as a single
+            DID controller group to control a DID document
+          " >
+          <figcaption>
+              Multiple <a>DID controllers</a> who are expected to act together as a
+              <a>DID controller</a> group
```
Comment (TallTed) [inline index.html:6302]:
```suggestion
Multiple <a>DID controllers</a> who are expected to act together as a
<a>DID controller</a> group
```

**Review by msporny — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -6221,7 +6221,9 @@ <h3>Set #1: The DID subject <em>is</em> the DID controller</h3>
 From a graph model perspective, even though the nodes identified as the
 <a>DID controller</a> and <a>DID subject</a> in figure 4 are distinct,
 there is a logical arc connecting them to express a semantic equivalence
-relationship (in RDF/OWL, this is expressed using the <a href="https://www.w3.org/TR/owl-ref/#sameAs-def"><code>owl:sameAs</code> predicate)</a>.
+relationship (in RDF/OWL, this is expressed using the <a
+href="https://www.w3.org/TR/owl-ref/#sameAs-def">
+<code>owl:sameAs</code> predicate)</a>.
```
Comment (TallTed) [inline index.html:6226]: "and then I read the text..."
```suggestion
relationship. (In RDF/OWL, this is expressed using the <a
href="https://www.w3.org/TR/owl-ref/#sameAs-def">
<code>owl:sameAs</code> predicate</a>.)
```

**Comment by msporny** [comment]
Merge conflicts, please fix.

**Review by peacekeeper — APPROVED** [review]

**Comment by rhiaro** [comment]
@msporny fixed conflicts

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**